cmd/cli/commands/deals.go

```diff
@@ -11,14 +11,14 @@ import (
 
 var (
 	dealsSearchCount  uint64
-	addToBlacklist    bool
+	blacklistTypeStr  string
 	crNewDurationFlag string
 	crNewPriceFlag    string
 )
 
 func init() {
 	dealListCmd.PersistentFlags().Uint64Var(&dealsSearchCount, "limit", 10, "Deals count to show")
-	dealCloseCmd.PersistentFlags().BoolVar(&addToBlacklist, "blacklist", false, "Add counterparty to blacklist")
+	dealCloseCmd.PersistentFlags().StringVar(&blacklistTypeStr, "blacklist", "none", "Whom to add to blacklist (worker, master or neither)")
```

Comment:
Properly describe valid flag values here: `neither` should be replaced with `none` as they parsed below.